src/api/users.js

```diff
@@ -41,14 +34,17 @@ export const all = (event, context, callback) => {
           },
         }).then((result) => {
           result = {data: result};
-          return done(null, result);
+          return callback(null, done(null, result));
         }).catch((error) => {
-          return done(error);
+          error.statusCode = 403;
```

Comment:
In other cases like this it should be 500 internal server error right?

Reply:
@singhabahu  http://restcookbook.com/HTTP%20Methods/400-vs-500/ is this what you mentioning?